src/test/java/utils/FixedNumbersGenerator.java

```diff
@@ -0,0 +1,18 @@
+package utils;
+
+import utils.generator.NumbersGenerator;
+
+import java.util.List;
+
+public class FixedNumbersGenerator implements NumbersGenerator {
+    private final List<Integer> numbers;
+
+    public FixedNumbersGenerator(List<Integer> numbers) {
+        this.numbers = numbers;
+    }
+
+    @Override
+    public List<Integer> generate() {
+        return numbers;
+    }
+}
```

Comment:
일반 로또 생성기에서는 필드 값이없는데 여기에는 있는 것은 부자연스럽다고 생각됩니다! 
```suggestion
public class FixedNumbersGenerator implements NumbersGenerator {

    @Override
    public List<Integer> generate() {
        return List.of(1,2,3,4,5,6);
    }
}
```
이렇게 수정하는건 어떨까요?

추가로 랜덤값에 대한 테스트가 어렵다고 하셨는데 잘 하시는 것 같습니다! 
하지만 테스트 각각에 여러가지 값을 테스트 해보고 싶으신 것 같은데 저는 이부분이 유닛 테스트와 통합 테스트와 관련 있다고 생각하빈다.

여러가지 값에 대한 테스트는 Lotto 객체에 대한 유닛 테스트라고 생각합니다. Lotto라는 단 하나의 객체에 대한 테스트에서 해당 검증을 마무리하고, 
numberGenerator를 사용하는 LottoService를 테스트하는 과정은 통합테스트죠 
Lotto에 들어간 값에 대해선 더이상 신경쓰지 않고 (이미 Lotto에 대한 테스트는 적절히 했다는 가정하에) 전체 로직의 흐름을 테스트 할 때 로또엔 고정된 값으로 랜덤에 대해 통제된 상황을 만드는 것이라고 생각합니다.

Reply:
단위 테스트와 통합테스트 각 테스트 에서 어떤 부분에 초점을 맞춰야할지를 생각 못하고 했던 고민이였던거 같습니다 
의견 감사합니다!